.travis.yml

```diff
@@ -13,8 +12,6 @@ matrix:
 notifications:
   email: false
 after_success:
-  - julia -e 'VERSION >= v"0.7.0-DEV.5183" && using Pkg;
```

Comment:
This should become
```
  - julia -e 'using Pkg;
```
(Check the travis log: This part fails, but it doesn't indicate a test failure.)